cli/start.go

```diff
@@ -146,13 +147,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	if context.EphemeralSingleNode {
 		// TODO(tamird): pass this to BootstrapRange rather than doing it
 		// at runtime. This was quicker, though.
```

Comment:
Yeah, except I still need to figure out what to do with this. It's currently triggering the race detector.
Once there's a system-wide default zone config, I'll be able to set it. Updating TODO